containers/Containerfile

```diff
@@ -1,9 +1,12 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest as builder
-RUN dnf install -y make git unzip gcc
+RUN dnf install -y make git unzip gcc openssl-devel
```

Comment:
I think LUAJIT2.*  is tightly coupled with openssl-* version.
If that's the case openssl-devel installation here will not be useful.

Reply:
It's actually required by wrk2 compilation process